addon/components/ember-tooltip-base.js

```diff
@@ -375,7 +380,14 @@ export default Component.extend({
 
       const { popperInstance } = this.get('_tooltip');
       const { popper } = popperInstance;
-      const marginSide = getOppositeSide(popper.getAttribute('x-placement'));
+      const side = popper.getAttribute('x-placement');
+
+      if (!side) {
+        // Bail out if popper hasn't finshed placement yet.
```

Comment:
I think what might be happening when the rAF callback is called and the tooltip is hidden (e.g. if it was queued up while still shown). We cancel it in `_hideTooltip`, but that might be too early, since the actual call to `_tooltip.hide()` happens in a later runloop. I'll change it up to cancel when we call `hide()` on the tooltip instance and see if the clears up the flakiness I was seeing in the test suite